metricbeat/module/system/services/_meta/fields.yml

```diff
@@ -0,0 +1,62 @@
+- name: services
+  type: group
+  release: beta
+  description: >
+    metrics for system services
+  fields:
+    - name: name
+      type: keyword
+      description: The name of the systemd service
+    - name: load_state
+      type: keyword
+      description: The load state of the unit
+    - name: active_state
+      type: keyword
+      description: The activity state of the service
+    - name: sub_state
+      type: keyword
+      description: The sub-state of the service
+    - name: exec_code
+      type: keyword
+      description: The SIGCHLD code from the service's main process
+    - name: exec_rc
+      type: long
+      description: The exit code of the service's main process
+    - name: state_since
+      type: date
+      description: The timestamp of the last state change. If the service is active and running, this is its uptime.
+    - name: main_pid
+      type: long
+      description: the main PID of the service.
+    - name: resources
+      type: group
+      description: system metrics associated with the service
+      fields:
+        - name: cpu.usage.nsec
+          type: long
+          description: CPU usage in nanoseconds
+        - name: memory.usage.bytes
+          type: long
+          description: memory usage in bytes
+        - name: tasks.count
+          type: long
+          description: number of tasks associated with the service
+        - name: network
+          type: group
+          description: network resource usage
+          fields:
+            - name: bytes.in
```

Comment:
network.in.bytes maybe? same below.

Reply:
Yah, there's a least one other metricset that uses `in.bytes`